src/VisualStudio/Core/Def/Implementation/UnusedReferences/Dialog/UnusedReferencesTableProvider.DataSource.cs

```diff
@@ -124,7 +124,9 @@ public bool TryGetValue(string keyName, out object? content)
                             break;
                         case UnusedReferencesTableKeyNames.ReferenceName:
                             // It is unnecessary to display the full path to project and assembly files.
-                            content = Path.GetFileNameWithoutExtension(ReferenceUpdate.ReferenceInfo.ItemSpecification);
+                            content = ReferenceUpdate.ReferenceInfo.ReferenceType == ReferenceType.Project
+                                ? Path.GetFileNameWithoutExtension(ReferenceUpdate.ReferenceInfo.ItemSpecification)
+                                : ReferenceUpdate.ReferenceInfo.ItemSpecification;
```

Comment:
The comment and the code are out of sync here. You're only trimming for project references, not assembly files

Reply:
Think I got it sorted now